libmpcdec 1.2.6 embeds mpc_decoder and does not provide mpc_decoder_destroy.

--- a/modules/demux/mpc.c
+++ b/modules/demux/mpc.c
@@ -246,7 +246,6 @@ static void Close( vlc_object_t * p_this )
     if( p_sys->p_demux )
         mpc_demux_exit( p_sys->p_demux );
 #else
-    mpc_decoder_destroy( &p_sys->decoder );
 #endif
     free( p_sys );
 }
